config/config-example.js

```diff
@@ -228,6 +228,11 @@ exports.repl = true;
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// disablehotpatchall - disables allowing hotpatching every hotpatchable part
+// of PS at once. Generally speaking, there probably won't be a need to for this,
+// which is why there's an option to disable this.
+exports.disablehotpatchall = false;
```

Comment:
I will admit, I'm a fan of documenting ``Config`` settings.  I don't really see any harm for it in this case.